install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -77,6 +77,11 @@ spec:
                 start of the range and the 2nd address represents the last usable address in the
                 range.
               type: string
+            provisioningRhcosImageURL:
```

Comment:
I've been thinking about this a bit, and I'm wondering if we should call this "MachineOSImage" instead.

The container itself does call it "RHCOS_IMAGE_URL" but we've largely moved on to referring to it as "Machine OS" image everywhere else, and I think new things should stick to that convention. 

Eventually, we will support baremetal IPI on OKD and that uses Fedora CoreOS instead of RHCOS.

Reply:
MachineOSImageURL? Do we want to have the URL in there still? makes it clearer in my mind..

Reply:
Yes. Makes sense to move away from a specific OS name and keep it generic. How do we differentiate this from OS image URL in the Machine CR though?

Reply:
Your proposal on slack `provsioningOSDownloadURL` is good, I like the idea to indicate in the name it's used in the downloader container, not directly by machines.

Reply:
Done.